### PR TITLE
Change the UI of the VC flow page

### PR DESCRIPTION
**Pull request by lmuntaner**
The main motivation of this change is to improve the UI of the verifiable credentials flow.

In this PR:
* Change UI for allowCredentials template.
* Change copy for allowCredentials page.
* Add two different showcases for the page, one with a simple consent message and one types of message.
* Add styles (for now used only in allowCredentials)

<hr/><details><summary>🟢 Some screens were added</summary></details><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2024-04-09 at 14 21 02

Tested with the demo issuer and test_app:

https://github.com/dfinity/internet-identity/assets/4550653/faa1a804-c802-4458-8881-18419416120e

**Comment by frederikrothenberger** [comment]
@lmuntaner: Could you also add a showcase page for the variant without anchor?

**Review by frederikrothenberger — APPROVED** [review]
E2e tests need fixing, but other than that it LGTM.
I assume this change has been discussed with Artem?
(Sorry, I'm a bit out of the loop)

**Comment by lmuntaner** [comment]
> I assume this change has been discussed with Artem?
Yes, sorry. I didn't explain it in the description. These changes were requested during the feedback we had with Jan and Sam about the VC playground. Artem designed this new screen.

> @lmuntaner: Could you change the showcase page for the variant without anchor to have less markdown text? Currently the anchor input is not actually _in_ the screenshot.

There are two showcases:
* The old one without anchor button and simple message.
* The new showcase with markdown and the anchor button.
allowCredentialsEdgecase

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-    "title": "Approve information sharing",
+    "title": "Present Credential",
     "allow_start": "Allow verifying credential",
     "label_ii_input": "Your Internet Identity",
     "allow_sep_with": "with",
     "cancel": "Cancel",
-    "allow": "Allow",
-    "issued_by": "Issued by",
-    "relying_party": "Relying party"
+    "allow": "Present Credential",
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply src/frontend/src/flows/verifiableCredentials/allowCredentials.json]: done

**Pull request merged**